contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/__init__.py

```diff
@@ -241,9 +242,12 @@ def _transmit_without_retry(self, envelopes):
 
 
 def new_metrics_exporter(**options):
-    options = Options(**options)
-    exporter = MetricsExporter(options=options)
-    transport.get_exporter_thread(stats.stats,
+    options_ = Options(**options)
```

Comment:
I don't have a strong opinion here, but I was curious on why do we need `options_` instead of using `options = Options(**options)`.

Reply:
It was to differentiate the parameter from the local variable. I didn't want to name it something completely different.